File: Elixir/fib/fib.py
from sys import argv
def fib(n):
    if n == 1 or n == 0:
        return n
    else:
        return fib(n-1) + fib(n-2)
    
if __name__ == "__main__":
    if len(argv) >= 2:
        print(fib(int(argv[1])))
    else:
        print('fib(n) | no input n')

File: Elixir/fib/fib.js
function fib(n) {
    if (n == 1 || n == 0) {
        return n
    }
    return fib(n - 1) + fib(n - 2)
}

if (process.argv && process.argv.length >= 3 && process.argv[2]) {
    // console.log(process.argv[2])
    console.log(fib(process.argv[2]))
} else {
    console.log('fib(n) | no input n')
}